src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/Blocks/payment_modules_list.html.twig

```diff
@@ -0,0 +1,68 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% trans_default_domain 'Admin.Payment.Feature' %}
+
+{% block payment_methods_module_list %}
+  {% if paymentModules|length > 0 %}
+  <div class="module-item-list">
+  {% for module in paymentModules %}
+    <div class="row module-item-wrapper-list border-bottom mb-sm-3">
+      <div class="col-12 col-sm-2 col-md-1 col-lg-1">
+        <div class="module-logo-thumb-list text-center">
+          <img src="{{ module.attributes.img }}" alt="{{ module.attributes.displayName }}"/>
+        </div>
+      </div>
+      <div class="col-12 col-sm-6 col-md-8 col-lg-9 pl-0">
+        <p class="mb-0">
+          {{ module.attributes.displayName|raw }}
+          <span class="text-muted">
+            {{ 'v%version% - by %author%'|trans({ '%version%' : module.attributes.version, '%author%' : module.attributes.author}, 'Admin.Modules.Feature') }}
+          </span>
+        </p>
+        <p class="text-muted">
+          {{ module.attributes.description|raw }}
+        </p>
+      </div>
+      <div class="col-12 col-sm-4 col-md-3 col-lg-2 mb-3">
+        <div class="text-center">
+          <a href="{{ path('admin_module_configure_action', {'module_name': module.attributes.name}) }}"
+             class="btn btn-primary-reverse btn-outline-primary light-button"
+          >
+            {{ 'Configure'|trans({}, 'Admin.Global') }}
+          </a>
+        </div>
+      </div>
+    </div>
+  {% endfor %}
+  </div>
+  {% else %}
+    <div class="alert alert-warning" role="alert">
+      <p class="alert-text">
+        {{ 'It seems there are no recommended payment solutions for your country.'|trans }} <br>
```

Comment:
Is this an error message?

Reply:
It's more like notification/warning. But not an error.

Reply:
If it's not a tooltip, I would say _Admin.Payment.Notification_ then!